src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -35,16 +36,17 @@ public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
             }
             //If the given file is a solution, get the list of projects
             //If not, then it's a project, which is put in a list
-            var projectsPaths = Path.GetExtension(listPackageArgs.Path).Equals(".sln")?
-                           MSBuildAPIUtility.GetProjectsFromSolution(listPackageArgs.Path).Where(f => File.Exists(f)):
+            var projectsPaths = Path.GetExtension(listPackageArgs.Path).Equals(".sln") ?
+                           MSBuildAPIUtility.GetProjectsFromSolution(listPackageArgs.Path).Where(f => File.Exists(f)) :
                            new List<string>(new string[] { listPackageArgs.Path });
 
             var autoReferenceFound = false;
+            var deprecatedFound = false;
 
             var msBuild = new MSBuildAPIUtility(listPackageArgs.Logger);
 
             //Print sources
-            if(listPackageArgs.IncludeOutdated)
+            if (listPackageArgs.IncludeOutdated)
```

Comment:
Can you illustrate the corner cases in the description? 

Like `--oudated` vs `--deprecated` vs both, and corner cases like --outdated only, but the latest package version is deprecated (but other, earlier version, are not?)

Reply:
See https://github.com/NuGet/Home/wiki/Deprecate-packages#cli